server/sonar-server/src/test/java/org/sonar/server/usergroups/ws/SearchActionTest.java

```diff
@@ -67,47 +68,47 @@ public void search_empty() throws Exception {
 
   @Test
   public void search_without_parameters() throws Exception {
-    insertGroup(defaultOrganizationProvider.getDto(), "users", 0);
-    insertGroup(defaultOrganizationProvider.getDto(), "admins", 0);
-    insertGroup(defaultOrganizationProvider.getDto(), "customer1", 0);
-    insertGroup(defaultOrganizationProvider.getDto(), "customer2", 0);
-    insertGroup(defaultOrganizationProvider.getDto(), "customer3", 0);
+    insertGroup(db.getDefaultOrganization(), "users", 0);
```

Comment:
you could store `db.getDefaultOrganization()` in a field and save on code duplication